### ISSUE TITLE
Enable eslint-plugin-jsx-a11y and fix existing issues

### ISSUE THREAD

**Issue by robertknight** [hypothesis/lms#1377]
As part of our general effort to improve accessibility and achieve WCAG compliance, introduce static linting to check for a11y errors in markup.

- [ ] Enable the jsx-a11y ESLint plugin for the lms frontend. Any existing failures should be flagged with a comment (`FIXME-A11Y`) and then temporarily suppressed
- [ ] Fix errors in `BasicLtiLaunchApp.js`
- [ ] Fix errors in `Dialog.js`
- [ ] Fix errors in `FileList.js`
- [ ] Fix errors in `LMSFilePicker.js`
- [ ] Fix errors in `StudentSelector.js`
- [ ] Fix errors in `ValidationMessage.js`

See also: The [corresponding issue](https://github.com/hypothesis/client/issues/1727) for the Hypothesis client.

**Comment by klemay** [comment]
@robertknight since all of these items are checked off, can we close this issue?
